packages/react-components/react-utilities/src/compose/overrideDefaultSlotClasses.ts

```diff
@@ -0,0 +1,11 @@
+import { mergeClasses } from '@griffel/react';
+import { SLOT_ORIGINAL_CLASS_NAME_SYMBOL } from '../compose/constants';
+import type { SlotComponentType, UnknownSlotProps } from '../compose/types';
+
+export const overrideDefaultSlotClasses = (slot: UnknownSlotProps, ...classNames: (string | false | undefined)[]) => {
```

Comment:
As discussed in tech sync, let's avoid a blessed class merging utility which could lead to confusion and abuse for the 99% of cases users should be using mergeClsases.

Let's just export a very obscure utiliity that returns the original class so that users make the explicit decision to take a dependency on it. This will buy us some breathing room to implement a long term next version of custom style hooks and also avoids a dependency on griffel